### PR TITLE
CYB-439 Merge daobet

### PR DESCRIPTION
**Pull request by adrianopol**

### CODE REVIEW

File: libraries/testing/include/eosio/testing/tester.hpp [hunk 1]
```diff
@@ -341,13 +341,13 @@ namespace eosio { namespace testing {
...
-         vcfg.reversible_cache_size = 1024*1024*8;
-         vcfg.reversible_guard_size = 0;
-         vcfg.contracts_console = false;
+         vcfg.blocks_dir            = tempdir.path() / std::string("v_").append(config::default_blocks_dir_name);
+         vcfg.state_dir             = tempdir.path() /  std::string("v_").append(config::default_state_dir_name);
+         vcfg.state_size            = 1024UL * 1024 * 1024;
+         vcfg.state_guard_size      = 1024UL * 1024 * 8;
+         vcfg.reversible_cache_size = 1024UL * 1024 * 1024;
+         vcfg.reversible_guard_size = 1024UL * 1024 * 8;
+         vcfg.contracts_console     = false;
```
Comment (adrianopol) [inline libraries/testing/include/eosio/testing/tester.hpp:350]: @justefg, pay your attention.

File: README.md [hunk 2]
```diff
@@ -1,53 +1,50 @@
-# Haya - blockchain with fast finality based on EOS
```
Comment (adrianopol) [inline README.md:1]: @justefg, pay your attention to changes in this file.

**Review by justefg — APPROVED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -45,7 +47,7 @@ script:
 deploy:
   provider: releases
   api_key:
-    secure: "jBbGdRNt1t8yYdvdYS/kDpzAZe32Xr4AbwJjnxyGKw3S3gsY64IV2GAEcpWnQA5+wPNprGc7YhcQZv8OfzlVoSA6EfEsFOe8ZIG1Qspq1dA+HqcrwxyOS2t0FcmOqdpmdsv+NXBpxO+RE0ukT5OrZO+2HM8dw9XPO8aEuAfKOAhSdW30blRzu/Yo2bBBIh/WA//TFKiOz/UrAv25Y0Jws6BXY/A8eJkP5KiAVnEsrSKpWOYZfXwUlVLxZ8edm1NCm/QIflO2K8CKTFYIFJ2YbdvCx1xE7soWIY4X3UxUea3svjax4/E/HI+qYmKoJPQzSLgOk21DzLH96MetO9TicPX9Bd5VhMHEcp76dLpEpcu1pS67v3u4nrFX8fw12xmZ40pvwAzUsng/e6QtFvudcQMNS/cSb+YsP4/akJkqg6vA0A66yMT8OY8AiPNfJZbiZA9a9waLmKGwjwYDdKkYDdssNdHmnUuzFpEDmESjo+VDIbTnglKwX5bq4BiGrDsdqgk93Qz4xY1VC3ItSX72iGIB74INlUiNQ9TDRVU66OTSqC9/XLSWZjwYWupXq2UpML0I47wtduIyEIdc/HcbQjqZRO/+L4/IduG0XBx5vlbTfLz+OpVPM9FxiJBvuIiOPNz1/O10WomMx5I0rNZOCDmeROR8OQgKs5siWA3ceoU="
```
Comment (algys) [inline .travis.yml]: leave old `api_key`, `api_key` is related to repository(1 by 1)

**Review by algys — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 3 above)
Comment (adrianopol) [reply .travis.yml]: Done.

**Review by algys — APPROVED** [review]

**Pull request merged**